test/jdk/javax/net/ssl/HttpsURLConnection/Equals.java

```diff
@@ -38,9 +38,24 @@
 
 public class Equals {
 
+    /*
+     * Enables the JSSE system debugging system property:
+     *
+     *     -Djavax.net.debug=ssl,handshake,record
+     *
+     * This gives a lot of low-level information about operations underway,
+     * including specific handshake messages, and might be best examined
+     * after gaining some familiarity with this application.
+     */
+    private static final boolean debug = false;
```

Comment:
Instead of hardcoding this, it's more flexible to use `Boolean.getBoolean("test.debug")` so you can run a test with debug output but not have to recompile the test.